index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Home Page</title>
+    <style>
```

Comment:
please update with some colors that aren't white or grey

Reply:
I have updated the colors as per your request. The body's background color is now purple (`#ab82ff`) and container's background color is now a light orange (`#ffc67d`). The updated colors should make the webpage more vibrant and visually appealing.